resources/monitoring/templates/prometheus/rules/alertmanager.rules.yaml

```diff
@@ -1,21 +1,21 @@
 {{- /*
-Generated from 'alertmanager.rules' group from https://raw.githubusercontent.com/coreos/kube-prometheus/release-0.1/manifests/prometheus-rules.yaml
+Generated from 'alertmanager.rules' group from https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/release-0.1/manifests/prometheus-rules.yaml
```

Comment:
If we decide to delete the dashboards, we can also delete all the rules in this folder. These are for Kubernetes 1.10 to 1.13.

Reply:
I talked to @a-thaler. He also agrees that we can delete them. @skhalash could you please delete this folder?